tests/Model/BufferTest.php

```diff
@@ -25,10 +25,11 @@ public function testEmpty(): void
 
     public function testFilled(): void
     {
-        $buffer = Buffer::filled(Area::fromScalars(0, 0, 10, 10), Cell::fromChar('X'));
+        $cell = Cell::fromChar('X');
+        $buffer = Buffer::filled(Area::fromScalars(0, 0, 10, 10), $cell);
         self::assertCount(100, $buffer);
         self::assertEquals(array_fill(0, 100, Cell::fromChar('X')), $buffer->content());
-        self::assertNotSame($buffer->get(Position::at(0, 0))->modifier, $buffer->get(Position::at(1, 1))->modifier, 'cells are propertly cloned!');
+        self::assertNotSame(spl_object_id($cell), spl_object_id($buffer->get(Position::at(1, 1))), 'cells are propertly cloned!');
```

Comment:
Same is an identity check anyway, so `spl_object_id` is redundant here I think

Reply:
Good catch.